### PR TITLE
feature: progress of pokemon details

### PR DESCRIPTION
**Pull request by JonathanAmador12**
add.
- retrieved data was shown in pokemon's details view

### CODE REVIEW

File: PokedexV2/Model/PokemonBasicModel.swift [hunk 1]
```diff
@@ -0,0 +1,26 @@
+//
+//  PokemonBasicModel.swift
+//  PokedexV2
+//
+//  Created by JonathanA on 19/10/22.
+//
+
+import Foundation
+struct PokemonBasicModel: Identifiable, Decodable, Hashable {
```
Comment (AmadorHeE) [inline PokedexV2/Model/PokemonBasicModel.swift]: PokemonBaseModel would be suitable in this case

File: PokedexV2/VIewModel/PokemonViewModel.swift [hunk 2]
```diff
@@ -9,10 +9,26 @@ import Foundation
 
 
 class PokemonViewModel: ObservableObject {
-    @Published var pokemon: [PokemonModel] = []
+    @Published var pokemon: [PokemonBasicModel] = []
+    @Published var pokemonDetails: PokemonModel?
```
Comment (AmadorHeE) [inline PokedexV2/VIewModel/PokemonViewModel.swift]: selectedPokemon is a better name

File: PokedexV2/Views/PokemonFeatures.swift [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+//  PokemonFeatures.swift
+//  PokedexV2
+//
+//  Created by JonathanA on 19/10/22.
+//
+
+import SwiftUI
+import Kingfisher
+
+struct PokemonFeatures: View {
```
Comment (AmadorHeE) [inline PokedexV2/Views/PokemonFeatures.swift]: Maybe PokemonDetailsView would be a suitable name

File: PokedexV2/Views/PokemonFeatures.swift [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+                        isActive = false
+                    }) {
+                        Image(systemName: "arrowshape.turn.up.backward.fill")
+                            .foregroundColor(Color("primary"))
+                    }
+                    .padding()
+                    Spacer()
+                }
+                ScrollView {
+                    if let pokemonDetail = vm.pokemonDetails{
```
Comment (AmadorHeE) [inline PokedexV2/Views/PokemonFeatures.swift]: Instead of pokemonDetails is pokemonDetails

File: PokedexV2/Views/PokemonFeatures.swift [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+                        VStack(spacing:20) {
+                            Text(pokemonDetail.name)
+                                .font(.title)
+                                .fontWeight(.heavy)
+                            KFImage(URL(string: pokemonDetail.imageUrl))
+                                .resizable()
+                                .aspectRatio(contentMode: .fit)
+                                .frame(width: 300, height: 300)
+                                .background(.red)
```
Comment (AmadorHeE) [inline PokedexV2/Views/PokemonFeatures.swift]: Why the color is always red ?

**Review by AmadorHeE — CHANGES_REQUESTED** [review]

**Review by AmadorHeE — APPROVED** [review]

File: PokedexV2/Model/PokemonBasicModel.swift (hunk 1 above)
Comment (JonathanAmador12) [reply PokedexV2/Model/PokemonBasicModel.swift]: it's done

File: PokedexV2/VIewModel/PokemonViewModel.swift (hunk 2 above)
Comment (JonathanAmador12) [reply PokedexV2/VIewModel/PokemonViewModel.swift]: it's done

File: PokedexV2/Views/PokemonFeatures.swift (hunk 3 above)
Comment (JonathanAmador12) [reply PokedexV2/Views/PokemonFeatures.swift]: it's done

File: PokedexV2/Views/PokemonFeatures.swift (hunk 4 above)
Comment (JonathanAmador12) [reply PokedexV2/Views/PokemonFeatures.swift]: it's done

File: PokedexV2/Views/PokemonFeatures.swift (hunk 5 above)
Comment (JonathanAmador12) [reply PokedexV2/Views/PokemonFeatures.swift]: I'm gonna do the next task

**Pull request merged**